### DISCUSSION TITLE
Failed Github Student Developer Pack!

### DISCUSSION THREAD

**Discussion by MiSu2002** [community/community#36912] [GitHub Education]
Hi everyone,

I had applied twice for Student Developer Pack but both of the times, the application got rejected. My college doesn't provide academic email address so, I provided all the needed documents for verification without academic email id. In first attempt, I uploaded my transcripts and on my second attempt, I provided the receipt from the bursar. But both of the applications got rejected demanding for academic email id. My college is hopeless and is not able to provide me an email address and I need the developer pack to create academic projects.

Please help me. It's urgent.

<img width="661" alt="Screenshot 2022-10-22 at 12 51 36 AM" src="https://user-images.githubusercontent.com/89409581/197274379-ff3102db-8eb1-48de-9263-823b05a9280e.png">
<img width="642" alt="Screenshot 2022-10-22 at 12 52 53 AM" src="https://user-images.githubusercontent.com/89409581/197274404-25936718-7e7e-4496-88ef-25633608a0f1.png">
<img width="864" alt="Screenshot 2022-10-22 at 12 54 15 AM" src="https://user-images.githubusercontent.com/89409581/197274407-868556f1-7991-42f5-81a6-26e7caafbb0c.png">

**Comment by segoSambel** [comment]
I faced the same problem yesterday, what I did was:
1. Add your academic email address to your github account
2. Cancel your latest github education request
3. Allow github.com to access your location
4. Re-apply github education request
5. Make sure you provide them with your valid document stating that you are an active student at your academic institution. like Student ID Card

**Reply by MiSu2002** [reply]
Thank you @segoSambel. The problem has been solved. I have provided University name and it worked.

**Reply by segoSambel** [reply]
Glad to hear that, enjoy your github education benefits @MiSu2002

Cheers

**Comment by kraack-tech** [comment]
Hi @MiSu2002,

Don't worry, there is still hope.

As stated [here](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/#i-tried-my-school-issued-email-but-im-still-being-asked-for-additional-proof) an academic email address is not a must - if your institution doesn't provide one. 

Follow this [guide ](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) for submitting additional proof.

**Reply by MiSu2002** [reply]
Thank you @kraack-tech. The problem has been solved. I have provided University name and it worked.

**Comment by neondev27** [comment]
I am living at Turkey, but i cant even do that, what i need to do?
